includes/payment-methods/class-wc-stripe-express-checkout-ajax-handler.php

```
@@ -89,52 +89,58 @@ public function ajax_add_to_cart() {
+				}
 			}
-		}
 
-		if ( ( ProductType::VARIABLE === $product_type || 'variable-subscription' === $product_type ) && isset( $_POST['attributes'] ) ) {
-			$attributes = wc_clean( wp_unslash( $_POST['attributes'] ) );
+			if ( ( ProductType::VARIABLE === $product_type || 'variable-subscription' === $product_type ) && isset( $_POST['attributes'] ) ) {
+				$attributes = wc_clean( wp_unslash( $_POST['attributes'] ) );
 
-			$data_store   = WC_Data_Store::load( 'product' );
-			$variation_id = $data_store->find_matching_product_variation( $product, $attributes );
+				$data_store   = WC_Data_Store::load( 'product' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Data_Store' in method 'ajax_add_to_cart'. (undefined)

(StaticAccess)